src/utilities.h

```diff
@@ -212,37 +212,38 @@ static const std::map<Variable::InterfaceType, const std::string> interfaceTypeT
     {Variable::InterfaceType::PUBLIC_AND_PRIVATE, "public_and_private"}};
 
 /**
- * @brief Convert the @p candidate @c std::string to a @c double.
+ * @brief Convert the @p from @c std::string to the @p to @c double.
```

Comment:
I very much prefer "input" and "output" (or something similar) than "from" and "to", just because it makes reading the documentation a little less mind-bending.  Very minor, and personal preference only though!

Reply:
Fair enough, I am now using `in` and `out` (shorter than `input` and `output` :)). Done in commit 94b2a7e2.